common/src/service/HttpClientHelper.cs

```diff
@@ -71,18 +73,19 @@ internal sealed class HttpClientHelper : IHttpClientHelper
                 _httpClientHandler.Proxy = customHttpProxy;
             }
 
+            _connectionLeaseTimeoutHandler = new ConnectionLeaseTimeoutHandler(connectionLeaseTimeout, _httpClientHandler);
```

Comment:
the `ConnectionLeaseTimeoutHandler` no longer seems to take in an http client handler, missed updating this one I guess

Reply:
Not sure how I missed this, thanks!